app/models/people/participant.rb

```diff
@@ -112,6 +117,10 @@ def method_missing(method_name, *args)
       super(method_name, *args)
     end
   end
+
+  def can_view?(object)
+    return true if object.id == id
```

Comment:
Actually, this needs to be a stricter check. Take a look at the `can_view` and `can_edit` methods for a `Mentor`, there should be a check for `object.is_a?(Participant)` before doing this comparison, because we don't know what the type of `object` might be.